woocommerce/woocommerce
# Add inventory section and SKU blocks

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Adding inventory section block along with SKU block and related logic.

Closes #37399 .



### How to test the changes in this Pull Request:


1. Enable the `block-editor-feature-enabled` feature flag. 
2. Browse to Products -> Add new
3. In the block editor, add a name in the name field and then click on the "Inventory tab"
4. You should see the Inventory section block with description and the SKU field
5. The SKU field should already be populated with a slug derived from the name that you added
6. Change the slug, and then save the product
7. Refresh the page to ensure the changes persist.
8. The name field should remain the same when you change the sku
